src/utils/tokenInfo.ts

```diff
@@ -97,3 +97,6 @@ export const getTokenSymbolWithHardcode = (
   if (chainId === ChainId.ARBITRUM && formatAddress === '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8') return 'USDC.e'
   return defaultSymbol ?? ''
 }
+
+export const getProxyTokenLogo = (logoUrl: string | undefined) =>
+  logoUrl ? `https://proxy.kyberswap.com/token-logo?url=${logoUrl}` : ''
```

Comment:
as said above, proxy is intend to use in kyberAI only. Need to re-design this

Reply:
I think the proxy feature is used for sharing KyberAI images with token logos. Here I also need proxy to do the same thing.
cc @nguyenhoaidanh